src/main/app/src/app/informatie-objecten/informatie-object-edit/informatie-object-edit.component.ts

```diff
@@ -274,8 +274,6 @@ export class InformatieObjectEditComponent implements OnInit, OnDestroy {
         console.log(key, value);
         if (value instanceof moment) {
           nieuweVersie[key] = value; // conversie niet nodig, ISO-8601 in UTC gaat goed met java ZonedDateTime.parse
-        } else if (key === "taal") {
```

Comment:
So the fix is that the language is no longer set automatically?

Reply:
It's mapped as is, in the `else` case, in stead of using the non-existing `value` property.